gwswlib/threedi.py

```diff
@@ -44,10 +44,44 @@ def __init__(self):
     def import_hydx(self, hydx):
         self.connection_nodes = []
         self.manholes = []
+        self.connections = []
+        self.structures = []
 
         for connection_node in hydx.connection_nodes:
+            check_if_element_is_created_twice(
+                connection_node.identificatieknooppuntofverbinding, self.connection_nodes, "Connection node"
+            )
             self.add_connection_node(connection_node)
 
+        for connection in hydx.connections:
+            check_if_element_is_created_twice(
+                connection.identificatieknooppuntofverbinding, self.connections, "Connection"
+            )
+
+            if connection.typeverbinding in ["GSL", "OPL", "ITR"]:
+                logger.warning(
+                    'The following "typeverbinding" is not connected in this importer: %s',
```

Comment:
De term "connected" is wat raar. Is "implemented" beter? Of "known"?
(Vooral ook omdat het bij de data om nodes en connecties gaat, dan klinkt "connection" als "deze node zit nergens aan vast").